hifi_vcpkg.py

```diff
@@ -82,23 +87,25 @@ def __init__(self, args):
 
         if 'Windows' == system:
             self.exe = os.path.join(self.path, 'vcpkg.exe')
-            self.bootstrapCmds = [ os.path.join(self.path, 'bootstrap-vcpkg.bat') ]
+            self.bootstrapCmds = [ os.path.join(self.path, 'bootstrap-vcpkg.bat'), '-disableMetrics' ]
             self.vcpkgUrl = self.assets_url + '/dependencies/vcpkg/builds/vcpkg-win32-client.zip%3FversionId=tSFzbw01VkkVFeRQ6YuAY4dro2HxJR9U'
             self.vcpkgHash = 'a650db47a63ccdc9904b68ddd16af74772e7e78170b513ea8de5a3b47d032751a3b73dcc7526d88bcb500753ea3dd9880639ca842bb176e2bddb1710f9a58cd3'
             self.hostTriplet = 'x64-windows'
-            if usePrebuilt:
-               self.prebuiltArchive = self.assets_url + "/dependencies/vcpkg/builds/vcpkg-win32.zip%3FversionId=3SF3mDC8dkQH1JP041m88xnYmWNzZflx"
+            #if usePrebuilt:
+            #    self.prebuiltArchive = "https://ipfs.io/ipfs/QmcBggttJQb1vYeyz29FXfaxnJ5c1HfZW72xNQepnENude/vcpkg-win32-a2623c6a-release.zip"
```

Comment:
Why comment out? Is the `usePrebuilt` value set wrongly somehow?
And why user IPFS?

Reply:
Use prebuilt can only be used when we have a valid archive. I've made my own archive and put it on Ipfs for testing that and ipfs in an earlier stage of the previous PR. Ipfs could make it cheaper for bandwidth usage, those archives are pretty big. And were even bigger before adding the feature to do release vckpg builds only.